server/controllers/bookController.js

```diff
@@ -7,18 +7,18 @@ const log = debug('dev');
 
 class BookController {
   static async addBook(req, res) {
-    const { title, description, amount, authorID, status, year, categoryID } = req.body;
+    const { title, description, amount, authorID, active, year, categoryID, isbn } = req.body;
     try {
       const book = await models.Books.create({
         title,
         description,
         categoryID,
         amount,
         authorID,
-        status,
-        year
+        active,
```

Comment:
What's the reason for the switch in name?

Reply:
The status was representing whether the book is **active** or not, and the value was a _boolean_. So I decided to switch the name to **active** so _boolean_ would make sense( **active**: _true_ ) better than ( **status**: _true_ )

Reply:
What do you mean active or not? Same as borrowed or not?

I feel it would have to be improved so we are able to store the status from an ENUM option.
The status could be either available, borrowed, under review etc.
Changing the active or inactive is limiting the options